src/SecureElementReader.App/Views/Titlebars/WindowsTitleBar.axaml.cs

```diff
@@ -0,0 +1,103 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+using Avalonia.Media;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace SecureElementReader.App.Views.Titlebars
+{
+    public partial class WindowsTitleBar : UserControl
+    {
+        private Button minimizeButton;
+        private Button closeButton;
+        private DockPanel titleBar;
+        private DockPanel titleBarBackground;
+        private TextBlock systemChromeTitle;
+        private NativeMenuBar seamlessMenuBar;
+        private NativeMenuBar defaultMenuBar;
+
+        public static readonly StyledProperty<bool> IsSeamlessProperty =
+        AvaloniaProperty.Register<WindowsTitleBar, bool>(nameof(IsSeamless));
+
+        public bool IsSeamless
+        {
+            get { return GetValue(IsSeamlessProperty); }
+            set
+            {
+                SetValue(IsSeamlessProperty, value);
+                if (titleBarBackground != null &&
+                    systemChromeTitle != null &&
+                    seamlessMenuBar != null &&
+                    defaultMenuBar != null)
+                {
+                    titleBarBackground.IsVisible = !IsSeamless;
+                    systemChromeTitle.IsVisible = !IsSeamless;
+                    seamlessMenuBar.IsVisible = !IsSeamless;
+                    defaultMenuBar.IsVisible = !IsSeamless;
+
+                    if (!IsSeamless)
+                    {
+                        titleBar.Resources["SystemControlForegroundBaseHighBrush"] = new SolidColorBrush { Color = new Color(255, 0, 0, 0) };
+                    }
+                }
+            }
+        }
+
+        public WindowsTitleBar()
+        {
+            this.InitializeComponent();
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) == false)
+            {
+                this.IsVisible = false;
+            }
+            else
+            {
+                minimizeButton = this.FindControl<Button>("MinimizeButton");
+                closeButton = this.FindControl<Button>("CloseButton");
```

Comment:
You should group all this FindControl functionality together. Is there a specific reason why they are separated in lines 57 and 58 and 65-69. 
Here we have FindControl, then two empty rows, then delegate assignments, then 2 empty rows and again FindControl